client/core/core.go

```diff
@@ -110,6 +160,8 @@ type Core struct {
 	wsConstructor func(*comms.WsCfg) (comms.WsConn, error)
 	userMtx       sync.RWMutex
 	user          *User
+	newCrypter    func(string) encrypt.Crypter
+	reCrypter     func(string, []byte) (encrypt.Crypter, error)
```

Comment:
Since every test of wallet-senstive functions requires checking the password = takes time, setting the `Crypter` constructor here allows a faster crypter to be used for testing.